app/src/main/java/comcmput301f16t01/github/carrier/RiderList.java

```diff
@@ -1,22 +0,0 @@
-package comcmput301f16t01.github.carrier;
-
-import java.util.ArrayList;
-
-import comcmput301f16t01.github.carrier.Users.User;
-
-/**
- * TODO: Remove/delete class when nothing references RiderList
```

Comment:
If you want me to do that... I should also comment everywhere for every method that I removed as well. These were marked as deprecated and have not been in our UML either. Could you tell me why you need me to post elsewhere?